docs/topics/review-reports.adoc

```diff
@@ -211,7 +211,7 @@ The image below shows the technologies used in the `jee-example-app`.
 .Technologies in an application
 image::technology-report-single-application.png[Technology report Application view]
 
-[id='application-dependencies-graph-report_{context}']
+[id="application-dependencies-graph-report_{context}"]
 === Application Dependencies Graph report

```

Comment:
I understand why you are capitalizing it this way but I think it looks funny. Although Dependencies Graph is part of the UI, there is a strong case for viewing it as generic. I recommend only capitalizing the first word.